build.gradle

```diff
@@ -165,6 +165,26 @@ publishing {
                 }
 
                 developers {
+                    developer {
+                        name = "Manabu Takayama"
+                        email = "learn.libra@gmail.com"
+                    }
+                    developer {
+                        name = "Toyama Hiroshi"
+                        email = "toyama0919@gmail.com"
+                    }
+                    developer {
+                        name = "Civitaspo"
+                        email = "civitaspo@gmail.com"
+                    }
+                    developer {
+                        name = "Shohei Sutoh"
+                        email = "ss.aquari@gmail.com"
+                    }
```

Comment:
@sutoh Hi!

As written in the pull-request description, we're trying to nominate developers who have contributed an important portion in `embulk-output-s3`.

We wonder if you're okay to nominate you in `pom.xml` of `embulk-output-s3` on Maven Central, from your work in #11. What do you think?

I picked up your name and email from the Git commit log and/or your public GitHub user page. Please let me know if you don't want, or if you want to be listed in another name/email.

Reply:
@dmikurube Thanks for the nominate. Yes, please use that name and email.